Code/GraphMol/MolHash/hashfunctions.cpp

```diff
@@ -463,12 +513,16 @@ std::string TautomerHashv2(RWMol *mol, bool proto, bool useCXSmiles,
   unsigned int hcount = 0;
   int charge = 0;
 
+  std::vector<std::uint64_t> atomFlags(mol->getNumAtoms(), 0);
```

Comment:
what's the intent behind `atomFlags`? Is there some deleted logic?

Reply:
> what's the intent behind `atomFlags`? Is there some deleted logic?

I figured that there is a non-zero chance that we'll need to incorporate logic blocking particular atoms at some point, so I just added it. They aren't currently doing anything

Reply:
thanks for clarifying! would be helpful to have a comment about that:)